.github/workflows/publish-docker-tannsi-relay.yml

```diff
@@ -0,0 +1,75 @@
+name: Publish Docker
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: tag (ex. v0.8.3) to publish on docker
+        required: true
+
+env:
+  BASE_URL: https://github.com/moondance-labs/tanssi/releases/download
+jobs:
+  tag-docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [
+          { name: "dancelight", file_name: "tanssi-relay" },
```

Comment:
```suggestion
          { name: "starlight", file_name: "tanssi-relay" },
```

Reply:
Isn't the runtime renamed to dancelight since https://github.com/moondance-labs/tanssi/commit/245235b1f2517142b315c69c63e13472d733b048 ?

Reply:
this is about the dockerhub repository, which I created under the name of starlight. and it's ok since we are publishing nodes here, not runtimes

Reply:
Sure,  updated in https://github.com/moondance-labs/tanssi/pull/696/commits/a53b28ef7be0fb8334f9fff484cd12f0e952fe9d